src/swagger.d.ts

```diff
@@ -1,106 +1,107 @@
 export interface Swagger {
-    swagger: string;
-    info: Info;
-    host?: string;
-    basePath?: string;
-    tags?: Tag[];
-    schemes: string[];
-    paths: Paths;
-    securityDefinitions?: { [key: string]: SecurityDefinition };
-    definitions?: { [key: string]: Definition };
-    externalDocs?: ExternalDocs;
+  swagger: string
+  info: Info
+  host?: string
+  basePath?: string
+  tags?: Tag[]
+  schemes: string[]
+  paths: Paths
+  securityDefinitions?: { [key: string]: SecurityDefinition }
+  definitions?: { [key: string]: Definition }
+  externalDocs?: ExternalDocs
 }
 
 // Info Section
 export interface Info {
-    title: string;
-    description?: string;
-    version: string;
-    termsOfService?: string;
-    contact?: Contact;
-    license?: License;
+  title: string
+  description?: string
+  version: string
+  termsOfService?: string
+  contact?: Contact
+  license?: License
 }
 export interface Contact {
-    name?: string;
-    url?: string;
-    email?: string;
+  name?: string
+  url?: string
+  email?: string
 }
 export interface License {
-    name: string;
-    url?: string;
+  name: string
+  url?: string
 }
 
 // Tag Section
 export interface Tag {
-    name: string;
-    description?: string;
-    externalDocs?: ExternalDocs;
+  name: string
+  description?: string
+  externalDocs?: ExternalDocs
 }
 
 // Path Section
 export interface Paths {
-    [key: string]: PathMethods;
+  [key: string]: PathMethods
 }
 
 type PathMethods = {
-    [method: string]: MethodDetails;
-};
+  [method: string]: MethodDetails
+}
 export interface MethodDetails {
-    tags?: string[];
-    summary?: string;
-    description?: string;
-    operationId: string;
-    consumes: string[];
-    produces: string[];
-    parameters: Parameter[];
-    responses: { [key: string]: Response };
-    security?: MethodSecurity[];
+  tags?: string[]
+  summary?: string
+  description?: string
+  operationId: string
+  consumes: string[]
+  produces: string[]
+  parameters: Parameter[]
+  responses: { [key: string]: Response }
+  security?: MethodSecurity[]
 }
 export interface Parameter {
-    in: string;
-    name: string;
-    description?: string;
-    required?: boolean;
-    schema?: SchemaRef;
+  in: string
+  name: string
+  description?: string
+  required?: boolean
+  schema?: SchemaRef
 }
 export interface SchemaRef {
-    $ref: string;
+  $ref: string
 }
 export interface Response {
-    description: string;
-    schema?: SchemaRef;
+  description: string
+  schema?: SchemaRef
 }
+
 export interface MethodSecurity {
-    [securityType: string]: string[];
+  [securityType: string]: string[]
 }
 
 // Security Section
 export interface SecurityDefinition {
-    type: string;
-    name?: string;
-    in?: string;
-    authorizationUrl?: string;
-    flow?: string;
-    scopes?: { [key: string]: string };
+  type: string
+  name?: string
+  in?: string
+  authorizationUrl?: string
+  flow?: string
+  scopes?: { [key: string]: string }
```

Comment:
Typescript now has the `Record` utility type so you don't have to do this
```suggestion
  scopes?: Record<string, string>
```

Reply:
Hm... This file wasn't eddited by me, i don't know why @SamWSoftware did it, but as i'm not a specialist in node/serverless-plugins and all the rest of code uses "[key: string]: string" maybe it's better to stay this way in my opnion

But thanks for the tip, i'll search about Record :)